DataFormats/BTauReco/interface/TemplatedSoftLeptonTagInfo.h

```diff
@@ -163,8 +163,8 @@ TaggingVariableList TemplatedSoftLeptonTagInfo<REF>::taggingVariables(void) cons
     const SoftLeptonProperties & data = m_leptons[i].second;
     list.insert( TaggingVariable(btau::leptonQuality,  data.quality(SoftLeptonProperties::Quality::leptonId, false)), true );
     list.insert( TaggingVariable(btau::leptonQuality2, data.quality(SoftLeptonProperties::Quality::btagLeptonCands, false)), true );
-    list.insert( TaggingVariable(btau::trackSip2dSig,  data.sip2d),    true );
-    list.insert( TaggingVariable(btau::trackSip3dSig,  data.sip3d),    true );
+    list.insert( TaggingVariable(btau::trackSip2dSig,  data.sip2d),    true ); //These are correct! it turns out is just a bad naming convention
```

Comment:
Please remove these comments. This (very old) inconsistency will be addressed in a separate PR (as announced in https://twiki.cern.ch/twiki/bin/view/CMS/CMSSW_7_6_0).